### ISSUE TITLE
Inconsistent HTML structure between checkbox-group and radio-group

### ISSUE THREAD

**Issue by wvteijlingen** [martndemus/ember-form-for#159]
It seems that each item in a radio-group is wrapped in an extra div, while this is not the case for items in a checkbox-group. I think it would be nice to have this consistent.

Radio group:
![screen shot 2017-01-19 at 17 18 29](https://cloud.githubusercontent.com/assets/1460535/22114927/5c2ca4dc-de6b-11e6-9789-b4dfc59c4217.png)

Checkbox group:
![screen shot 2017-01-19 at 17 18 39](https://cloud.githubusercontent.com/assets/1460535/22114930/5f30d126-de6b-11e6-85e2-b51a0f7e2de8.png)

**Comment by marxsk** [comment]
I agree that this should be done. The reason is that we use standard radio-field inside of group but we options are handled differently. This should be fixed after moving away from ember-one-way-controls.
